scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -275,10 +271,11 @@ object BigQueryTypedTable {
    * Writting: Supports LogicalTypes only for DATE and TIME.
    *           DATETIME is not yet supported. https://issuetracker.google.com/issues/140681683
    */
-  def apply(table: Table, format: Format): BigQueryTypedTable[format.F] = format match {
-    case Format.GenericRecord => genericRecord(table).asInstanceOf[BigQueryTypedTable[format.F]]
-    case Format.TableRow      => tableRow(table).asInstanceOf[BigQueryTypedTable[format.F]]
-  }
+  def apply[F: Coder](table: Table, format: Format[F]): BigQueryTypedTable[F] =
```

Comment:
Is `format` a required argument now? Do we want to make it an implicit and provide a default, e.g. `TableRow` and later switch to Avro?

Reply:
It's only required if users choose to use the IO directly coz otherwise:
https://github.com/spotify/scio/blob/b78dd27ea8caf196cdc7d03dea4862fc4ce92bf9/scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala#L73-L87

imo I don't think we would gain by having it implicitly declared.